ios/ReactNativePayments.m

```diff
@@ -321,7 +322,7 @@ - (NSArray *_Nonnull)getSupportedNetworksFromMethodData:(NSDictionary *_Nonnull)
         }
     }
 
-    return shippingMethods;
+    return nil;
```

Comment:
Obviously we can't do this. See if we actually need this, perhaps we only return nil based on request shipping option? do we actually have to return nil to not have shipping show up in the dialog?